src/styles/components/jText.m.scss

```diff
@@ -7,6 +7,10 @@
   transition: color 0.3s ease 0s, opacity 0.3s ease 0s, font-size 0.3s ease 0s;
 }
 
+.main-black {
```

Comment:
Why is JText still alive?

Reply:
I missed the moment when he was completely abandoned. Even from using its classes